README.md

```diff
@@ -77,7 +77,7 @@ You can sort/filter/download the data from [Airtable](https://airtable.com/shrXv
 ## J - L
 - [Janaki Tech](http://janakitech.com/) | Harihar Bhawan, Lalitpur | We strive to build better software useful for everyone
 - [Javra](http://javra.com/) | Kumaripati, Lalitpur | IT solutions that boost productivity in business
-- [Jyaasa](http://jyaasa.com/) | Jana Bahal Rd, Patan | We Design, Build, and Develop Products
+- [Jyaasa](http://jyaasa.com/) | Imadol, Lalitpur | At Jyaasa we transform ideas into world class products. Our professional team is fluent in 3Rs (Ruby on Rails, React and React Native). We’re ready to design, build and develop something that your users will love. 
```

Comment:
@gkunwar Thanks for the update! 

Can we shorten this description to two lines? Also, please remove the full stop in the end.

Here are the guidelines: https://github.com/mesaugat/tech-companies-in-nepal#add-your-company-to-the-list

Reply:
Thank you @mesaugat for your review and feedback. I will update PR and let you know.